### DISCUSSION TITLE
Connection pool error - how to configure this in mirth

### DISCUSSION THREAD

**Discussion by daniel-ls-howard** [nextgenhealthcare/connect#5922] [Q&A]
Hi All,

I saw this on our DEV server. I think this is coming from mirth connecting to it's own DB.   Is there settings to specify for the mirth internal connection pool?

	at org.quartz.simpl.SimpleThreadPool$WorkerThread.run(SimpleThreadPool.java:557)Caused by: com.mysql.cj.exceptions.CJCommunicationsException: 

	The last packet successfully received from the server was 158,789,975 milliseconds ago.  
	The last packet sent successfully to the server was 158,789,979 milliseconds ago. is longer than the server 
	configured value of 'wait_timeout'. You should consider either expiring and/or testing connection 
	validity before use in your application, increasing the server configured values for client timeouts, 
	or using the Connector/J connection property 'autoReconnect=true' to avoid this problem.

**Comment by ab-mg-23** [comment]
Assuming it is the internal connections, set the `database.test-query` property in `mirth.properties` and bounce the service. The default configuration uses HikariCP and will then use the configured test query when pulling connections from the pool.
